src/Native/System.Security.Cryptography.Native/openssl.c

```diff
@@ -339,17 +340,24 @@ GetAsn1StringBytes(
     unsigned char* pBuf,
     int cBuf)
 {
-    if (!asn1)
+    if (!asn1 || cBuf < 0)
```

Comment:
I think so, but invalid args are handled without asserting throughout this file. I'm looking at crypto shims now and I'll clean this up as part of that.